### PR TITLE
Add text to Overview page

### PR DESCRIPTION
**Pull request by anibalsolon**
Hello dear @PeerHerholz @SNeuroble @shawnrhoads @htwangtw (and everyone with some energy for a text review!)

This is a proposal from @katjaq and yours truly for the Overview page, giving a brief... overview of the book. The figure name is wrong at the moment, but I believe it will be fixed by another PR. 🙃

Please give us your comments!

### CODE REVIEW

**Review by htwangtw — APPROVED** [review]
We have an overview page with content :partying_face: 
It's possible to switch the URL to each section to an internal link using labels
See [jupyterbook doc](https://jupyterbook.org/content/citations.html#reference-section-labels)
I made a PR to this branch in @anibalsolon 's fork.

**Comment by katjaq** [comment]
You're the best @htwangtw ! Thanks so much!! 🚀

File: brainhack_book/overview.md [hunk 1]
```diff
@@ -11,3 +11,23 @@ kernelspec:
 ---
 
 # Overview
+
+The Brainhack Jupyter book has been written by the Brainhack community, as a continuously growing resource for the past and future of Brainhacks. Built together, it documents the achievements and organization of previous events around the world, and aims to instigate events and initiatives towards an open, inclusive, and community-driven neuroscience.
+
+Not subject to the same space restrictions as the NeuroView, the book can become a place where all our ideas can live, illustrated with several projects reflecting the colorful range of interests across the entire community.
```
Comment (Remi-Gau) [inline brainhack_book/overview.md:17]:
```suggestion
Not subject to the same space restrictions as a traditional journal article, the book can become a place where all our ideas can live, illustrated with several projects reflecting the colorful range of interests across the entire community.
```

**Comment by katjaq** [comment]
Thank you @Remi-Gau for going through and improving & approving the text ✨ and thanks for implementing already the suggested changes @htwangtw 🚀

**Review by PeerHerholz — COMMENTED** [review]
Thx for this! I was wondering about two things: 1) I assume you tested it, but do the links to the sections work like that and 2) some sort of reference/information for `jupyter book` and `binder` could/should be added given that without these two initiatives this whole thing wouldn't be possible. WDYT?

**Comment by htwangtw** [comment]
> 1) I assume you tested it, but do the links to the sections work like that and 

Worked on a local build that uses the same environment. 
Thought related to this: we can add a minimal book build action on PRs to automatically check this kind of things? I don't know how to set it up exactly but that would be handy.

>2) some sort of reference/information for jupyter book and binder could/should be added given that without these two initiatives this whole thing wouldn't be possible. 

Love this idea and we should totally do it!

**Comment by anibalsolon** [comment]
Thank you all @htwangtw @Remi-Gau @PeerHerholz for the feedbacks/updates! They look great ✨

About Peer's points: 1) it seems like @htwangtw updated already! 💥  and 2) yep, definitely!

About setting an automagic links `unit test`, I can take a look at that. Maybe a headless browser that goes clicking in every link to see if it works, and check all the images.

**Review by SNeuroble — APPROVED** [review]

**Review by SNeuroble — APPROVED** [review]

File: brainhack_book/overview.md [hunk 2]
```diff
@@ -11,3 +11,23 @@ kernelspec:
 ---
 
 # Overview
+
+The Brainhack Jupyter book has been written by the Brainhack community, as a continuously growing resource for the past and future of Brainhacks. Built together, it documents the achievements and organization of previous events around the world, and aims to instigate events and initiatives towards an open, inclusive, and community-driven neuroscience.
+
+Not subject to the same space restrictions as a traditional journal article, the book can become a place where all our ideas can live, illustrated with several projects reflecting the colorful range of interests across the entire community.
+
+Over the next few months, we aim to write the book as a combination of elements from the short paper, the initial section drafts, the large pool of ideas from our brainstorming sessions, and the great variety of projects, born in Brainhacks and led by the team effort of numerous Brainhackers. Please join our efforts <3
```
Comment (SamGuay) [inline brainhack_book/overview.md:19]:
```suggestion
Over the next few months, we aim to write the book as a combination of elements from the short paper, the initial section drafts, the large pool of ideas from our brainstorming sessions, and the great variety of projects, born in Brainhacks and led by the team effort of numerous Brainhackers. Please join our efforts <3.
```

**Review by SamGuay — APPROVED** [review]

File: brainhack_book/overview.md (hunk 2 above)
Comment (anibalsolon) [reply brainhack_book/overview.md:19]: Thank you! Maybe `Please join our efforts! <3` ? 🙃

Comment (SamGuay) [reply brainhack_book/overview.md]: Up to you, I didnt like that the sentence didn't end :upside_down_face: .

**Review by SNeuroble — APPROVED** [review]

**Pull request merged**